### PR TITLE
Update docs/index.md to include correct query header

### PR DESCRIPTION
**Pull request by pdecat**

### CODE REVIEW

**Review by misraved — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ og_image: "/images/plugins/turbot/code-social-graphic.png"
 
 Source code can be any string or data for querying.
 
-[Steampipe](https://steampipe.io) is an open source CLI to instantly query cloud APIs using SQL.
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query local data or APIs (cloud or not) using SQL.
```
Comment (misraved) [inline docs/index.md:17]: Thanks @pdecat for making the changes, however, we have been using this description across all our plugins(agreed we have gone beyond just cloud APIs).

@cbruno10 do you think we should update all our plugins to use the following description - 
```
[Steampipe](https://steampipe.io) is an open source CLI to instantly query local data or APIs (cloud or not) using SQL.
```

Comment (cbruno10) [reply docs/index.md:17]: @misraved Do we have any other messaging on steampipe.io, or in other marketing material, that describes Steampipe?

Comment (misraved) [reply docs/index.md:17]: @cbruno10 currently we don't have any marketing material that describes Steampipe. 

This [introductory blog](https://steampipe.io/blog/introducing-steampipe) only talks about using Steampipe for Cloud APIs though.

**Comment by cbruno10** [comment]
Hey @pdecat , sorry for the long response time on this PR. We agree that the current line we use in our plugin and mod documents describing Steampipe is now outdated, but admittedly we're still working on what that line should be across all of our channels.

For now, we'll close this PR as we continue to work on our common Steampipe description, but I have made the update to the example description in https://github.com/turbot/steampipe-plugin-code/commit/89a942403c46c90eeaf7cf5e90f8ea3186482dc1 (slightly different wording) that you pointed out.

**Pull request closed without merging**